run_dir/design/agreements_tab.html

```diff
@@ -9,5 +9,6 @@
   <script src="/static/js/marked.min.js"></script>
   <script src="/static/js/pricing_main.js?v={{ gs_globals['git_commit'] }}"></script>
   <script src="/static/js/pricing_quote.js?v={{ gs_globals['git_commit'] }}"></script>
+  <script src="/static/js/running_notes.js?v={{ gs_globals['git_commit'] }}" id="rn-js" data-project="{{project}}"></script>
```

Comment:
You don't need this here, its already included in the main page

Reply:
Great!